cc_proxy.go

```diff
@@ -205,10 +203,21 @@ func (p *ccProxy) sendCmd(cmd interface{}) (interface{}, error) {
 		return nil, fmt.Errorf("Wrong command type, should be hyperstartProxyCmd type")
 	}
 
-	err := p.client.Hyper(proxyCmd.cmd, proxyCmd.message)
+	data, err := json.Marshal(proxyCmd.message)
 	if err != nil {
 		return nil, err
 	}
 
+	var tokens []string
+	if proxyCmd.token == "" {
+		tokens = nil
+	} else {
+		tokens = append(tokens, proxyCmd.token)
+	}
+
+	if err := p.client.HyperWithTokens(proxyCmd.cmd, tokens, json.RawMessage(data)); err != nil {
```

Comment:
This function could just end with:

```
return nil, p.client.HyperWithTokens(proxyCmd.cmd, tokens, json.RawMessage(data))
```

But that makes me wonder, why does this function always return a `nil` `interface`? Do we envisage a proxy that might not return this value at some point? If not, why don't we just return an `error`?